editor/bundle/src/main/java/io/wcm/caconfig/editor/model/EditorConfiguration.java

```diff
@@ -56,11 +60,13 @@
   @Inject
   public EditorConfiguration(@SlingObject Resource currentResource,
       @OSGiService ConfigurationResourceResolver configResourceResolver,
-      @OSGiService EditorConfig editorConfig) {
+      @OSGiService EditorConfig editorConfig,
+      @AemObject Page currentPage) {
     this.configNamesUrl = currentResource.getPath() + "." + ConfigNamesServlet.SELECTOR + ".json";
     this.configDataUrl = currentResource.getPath() + "." + ConfigDataServlet.SELECTOR + ".json";
     this.configPersistUrl = currentResource.getPath() + "." + ConfigPersistServlet.SELECTOR + ".json";
     this.contextPath = configResourceResolver.getContextPath(currentResource);
+    this.language = currentPage.getLanguage(false).getLanguage();
```

Comment:
I have no idea if this is the best place to put this. Seemed like the quickest/easiest option.